src/mappings/util/metadata.ts

```diff
@@ -36,7 +36,11 @@ export async function fetchMetadata(url: string, job: Queue.Job) {
     try {
         const { status, data } = await api.get(finalUrl)
         if (status < 400) {
-            return data
+            if (data && typeof data === 'object' && !Array.isArray(data)) {
+                return data
+            }
+            job.log(`Invalid response from ${url}`)
+            job.log(data)
         }
```

Comment:
More verbosity, let's get the status code logged on-error.

```suggestion
        if (status >= 200 && status < 300) {
            if (data && typeof data === 'object' && !Array.isArray(data)) {
                return data
            }

            job.log(`Invalid response from ${url}`)
            job.log(data)
        } else {
            job.log(`Status code ${status} received from ${url}`)
        }
```